frontend/src/components/Common/SwipeCard.js

```diff
@@ -0,0 +1 @@
+import ',/swipeCard.css';
```

Comment:
What is the purpose of this? it seems like a file with a single line....?

Reply:
SwipeCard.(js/css) is to make the card reusable but I havent quite gotten to it yet. Should i just delete those?

Reply:
Is it currently a place holder right now?

Reply:
Yes

Reply:
I would think it would be fine then. If its just a placeholder for future work, it should be fine to leave it in.

Reply:
Cool, If anyone has differing opinions, let me know thanks

Reply:
I think the card should be componentised so that it can be used in both SwipingPage and the results page